.jenkins/pipelines/release.Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent {
+        label 'Jenkins-Shared-DC2'
+    }
+    options {
+        timeout(time: 300, unit: 'MINUTES')
+    }
+    parameters {
+        string(name: "REPOSITORY", defaultValue: "deislabs")
+        string(name: "BRANCH", defaultValue: "main", description: "Branch to build")
+        choice(name: "REGION", choices:['useast', 'canadacentral'], description: "Azure region for the SQL solutions test")
+        string(name: "PACKAGE_NAME", description: "optional - release package to install (do not include extension)")
```

Comment:
Is this really optional? If it is you need to put a defaultValue.
Also, what tests are ran when PACKAGE_NAME is not defined, in the context of the matrix and the TEST_PACKAGE axis?

Reply:
Good catch, `PACKAGE_NAME` isn't optional in this pipeline